### DISCUSSION TITLE
Enhancement - add pac webresource push!

### DISCUSSION THREAD

**Discussion by dmcweeney** [microsoft/powerplatform-vscode#661] [Ideas]
Hi,

We need a way to automate the deployment of webresource builds to an environment.
The pac tool would seem the ideal home for such a feature.

The current preview feature of shared librarys in pcf components is something that would be very useful for webresources - having webresource management built into the pac tool would help achieve this.

Thanks, Donal

**Comment by filcole** [comment]
@dmcweeney see this discussion https://github.com/microsoft/powerplatform-vscode/discussions/326, which I think is relevant here.
